lib/win-rebase.js

```diff
@@ -1,8 +1,17 @@
 var re = /^\s*("*)([^"]*?\b(?:node|iojs)(?:\.exe)?)("*)( |$)/
+var npmre = /^\s*("*)([^"]*?\b(?:npm))("*)( |$)/
+var path_ = require('path')
+if (path_.win32) path_ = path_.win32
 
-module.exports = function (path, rebase) {
+module.exports = function (path, rebase, whichOrUndefined) {
```

Comment:
I'd be tempted to pull `whichOrUndefined` into its own little helper module (like `isWindows`) rather than passing it as an argument.